reference/forms/types/options/label_format.rst.inc

```diff
@@ -0,0 +1,38 @@
+label_format
+~~~~~~~~~~~~
+
+.. versionadded:: 2.6
+    The ``label_format`` option was introduced in Symfony 2.6.
+
+**type**: ``string`` **default**: ``null``
+
+Sets the format used to generate the ``<label>`` element. This is usefull when
```

Comment:
`usefull` -> `useful` (English language should consider start accepting `usefull` because I know nobody who writes it correctly)